README.md

```diff
@@ -26,3 +26,5 @@ Please complete the following by November 1st. This should be the next time we w
 * Trey Watford is Bester
 * Will Dixon Is better
 * Chris Bracey Is betterer
+* Kyle Heath Is bettererer
```

Comment:
@DylanLeon  there should alost be a lie starting with >>>>>>>>>>>>>>>>>>> (branch name) be sure to remove that and any duplicated data